.github/workflows/operator-ci.yml

```diff
@@ -1,6 +1,13 @@
 name: Keycloak Operator CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore: [main]
+  pull_request:
+    # paths-ignore:
```

Comment:
@andreaTP please, feel free to remove those comments. I don't think we need them.

Reply:
hum, if i may ask: why? if it works, this wouldn't start the whole CI if only .md files are changed. we could actually use the same approach for *.adoc files, so the upcoming docs changes should not exhaust the CI too much. Nevertheless can surely be handled in a different commit / ticket :)

Reply:
@DGuhr by removing those comments I mean, deleting those lines. We don't have any plans to use those lines. My apologies, bad wording :)

Reply:
I thought we wanted to start going this direction, anyhow, removed